RecoLocalTracker/SiPixelRecHits/plugins/SiPixelRecHitSoAFromLegacy.cc

```diff
@@ -77,7 +81,7 @@ void SiPixelRecHitSoAFromLegacy::produce(edm::StreamID streamID, edm::Event& iEv
     throw cms::Exception("Configuration") << "SiPixelRecHitSoAFromLegacy can only use a CPE of type PixelCPEFast";
   }
   auto const& cpeView = fcpe->getCPUProduct();
-
+  //auto maxInModule = 0U, maxDigi = 0U;
```

Comment:
remove altogether?

Reply:
Yes, will make a cleanup in the next commit.